components/MainNavBarItem.vue

```diff
@@ -3,11 +3,11 @@
     <a :href="url" class="block mt-4 xl:pt-1 xl:pb-1 xl:inline-block lg:mt-0 text-gray-500 hover:text-black lg:mr-2 xl:mr-4" v-if="external">
       <slot></slot>
     </a>
-    <nuxt-link :to="url" class="block xl:pt-1 xl:pb-1 mt-4 lg:inline-block lg:mt-0 text-gray-500 hover:text-black lg:mr-2 xl:mr-4" :class="{ 'menuActive' : isActive}" v-else>
+    <nuxt-link :to="url" class="block xl:pt-1 xl:pb-1 mt-4 lg:inline-block lg:mt-0 text-gray-500 hover:text-black lg:mr-2 xl:mr-4" :class="{ 'menuActiveBar' : isActive || isSubpageActive}" v-else>
```

Comment:
Not really an error, but I would probably change the name from menuActiveBar to menuBarItemActive. Just makes it a bit clearer about what is going on. But being very nit-picky.

You will also need to change the name in the css below.

Thanks